docs/en_us/platform_api/source/courses/blocks.rst

```diff
@@ -0,0 +1,325 @@
+.. _Courses API Blocks Resource:
+
+########################################
+Courses API Blocks Resource
+########################################
+
+With the Courses API **Blocks** resource, you can complete the
+following tasks.
+
+
+.. contents::
+   :local:
+   :depth: 1
+
+.. _Get a list of the course blocks in a course:
+
+****************************************
+Get a List of Course Blocks in a Course
+****************************************
+
+The endpoint to get a list of course blocks in a course is
+``/api/courses/v1/blocks/`` with required ``course_id`` parameter.
+
+=====================
+Use Case
+=====================
+
+Get a list of course blocks in a specified course. Response results depend on
+the requesting user's access level.
```

Comment:
@nasthagiri Can you pls clarify? What is the "access level" check made against? Is it related to course admin permissions or something else?